.travis.coverage.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+if [ "$TRAVIS_PHP_VERSION" = '7.1' ] ; then
+    wget https://scrutinizer-ci.com/ocular.phar
+    php ocular.phar code-coverage:upload --format=php-clover ./clover.xml
```

Comment:
`clover.xml` will not be magically generated, we need to pass the flags to PHPUnit